AfricanReligions/interface/generalstuff.gfx

```diff
@@ -1275,4 +1275,10 @@ spriteTypes = {
 		noOfFrames = 6
 		allwaystransparent = yes
 	}
+	
+	#DECISIONS
+	spriteType = {
+		name = "GFX_pick_religious_tradition"
+		texturefile = "gfx\\interface\\decision_icon_pick_main_deity.dds"
+	}
```

Comment:
Let's start a new file for decision pictures.  I should probably have made some decision pictures for the other decisions, haha.